app/components/data/shared/cloud-file-picker/cloud-file-picker-dialog.component.ts

```diff
@@ -56,8 +59,14 @@ export class CloudFilePickerDialogComponent {
         });
     }
 
-    public updatePickedFile(file: string) {
-        this.pickedFile = file;
+    public ngOnInit() {
+        if (this.wildcards) {
+            this.pickedFilter = this.wildcards;
+            this.optionFilters = [
+                { label: "All Files", value: "" },
+                { label: `Wildcards (${this.wildcards})`, value: this.wildcards },
```

Comment:
I would maybe not say Wildcards in front just the file extension

Reply:
What about "(*.blend,*.other) Files". Or just leave out the "Files" as well.

Reply:
Yeah files would work too I feel